hhs_hosp/delphi_hhs/run.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""Functions to call when running the function.
+
+This module should contain a function called `run_module`, that is executed
+when the module is run with `python -m delphi_hhs`.
+"""
+from datetime import date, datetime, timedelta
+
+from delphi_epidata import Epidata
+from delphi_utils import read_params
+from delphi_utils.export import create_export_csv
+from delphi_utils.geomap import GeoMapper
+import pandas as pd
+
+from .constants import HOSPITALIZATIONS
+
+def int_date_to_previous_day_datetime(x):
+    """Convert integer dates to Python datetimes for the previous day.
+
+    Epidata uses an integer date format. This needs to be converted to
+    a datetime so that the exporter can interpret it.
+
+    The HHS columns we are interested in measure admissions for the
+    previous day. To accurately indicate the date of incidence, the
+    date must be shifted back by one day.
+    """
+    return x.apply(lambda x: datetime.strptime(str(x), "%Y%m%d") - timedelta(days=1))
+
+def run_module():
+    """Generate ground truth HHS hospitalization data."""
+    params = read_params()
+    mapper = GeoMapper()
+
+    # get a list of state codes
+    all_states = pd.DataFrame(
+        { "state_code": ["{:02d}".format(x) for x in range(1, 72)] }
```

Comment:
Is there a simple way to get only state codes that actually exist? The line `all_states =  mapper.add_geocode(all_states, "state_code", "state_id")` removes the codes that don't exist, so not a big deal.

Reply:
might be useful to have the geomap util contain useful constants